Makefile

```diff
@@ -165,7 +165,8 @@ lint-gosec:
 .PHONY: lint-staticcheck
 lint-staticcheck:
 	@cat /dev/null > $(LINT_LOG)
-	time staticcheck ./... \
+	# TODO remove SA1019 once https://github.com/golang/protobuf/issues/1077 is resolved
+	time staticcheck -checks=-SA1019 ./... \
```

Comment:
Instead of ignoring `SA1019` universally does it make more sense to add the offending files to the list below?
```
		| grep -v \
			-e model/model.pb.go \
			-e model/prototest/model_test.pb.go \
			-e thrift-gen/ \
```

The other two files (cmd/collector/app/zipkin/http_handler_test.go, model/ids_test.go) are not autogenerated and could have `nolint` directives added. Although to keep things in one spot, might be better to add all three here with a note to remove once https://github.com/golang/protobuf/issues/1077 is resolved.

Reply:
This does not seem to be needed after I removed proto version bump. I will remove it for now. 

We will run into this when upgrading to the newer proto.